variables.tf

```diff
@@ -344,3 +344,52 @@ variable "s3_lifecycle_policy_transition_period" {
   type        = "string"
   default     = "30"
 }
+
+variable "rds_family" {
```

Comment:
If this is for HMS, shouldn't we use HMS as a prefix in the variable name? This name is too generic.